src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs

```diff
@@ -324,16 +326,23 @@ public async Task<IActionResult> FileUploadReview(FileUploadReviewViewModel view
                 case FileUploadReviewOption.ApproveAndSend:
                     throw new NotImplementedException();
                 case FileUploadReviewOption.SaveButDontSend:
-                    var apiRequest = await _modelMapper.Map<BulkUploadAddDraftApprenticeshipsRequest>(viewModel);
-                    await _commitmentApiClient.BulkUploadDraftApprenticeships(viewModel.ProviderId, apiRequest);
-                    TempData.AddFlashMessage("File uploaded", ITempDataDictionaryExtensions.FlashMessageLevel.Success);
-                    return RedirectToAction(nameof(Review), new { ProviderId = viewModel.ProviderId });
-                    
+                    return RedirectToAction("SuccessSaveDraft", viewModel);                    
```

Comment:
should the api post  be done here and then only redirect on successful response?
Then the SuccessSaveDraft would just be responsible for playing back rather than the submission also